app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -13,6 +13,10 @@ import '@numbers/sleeping-blocks';
 import { withLocalMutation, withLocalState } from '@numbers/ducks';
 
 const blockTypes = { githubPull, githubCommit, slackMessage };
+const wallWidth = 50;
+
+// Padding applied to the height of the walls to prevent blocks created contemporaneously from colliding.
```

Comment:
not sure I understand this comment. Why are blocks colliding with walls and why was it a problem?

Reply:
The theory was there was this edge case where blocks could be created close to inside one another and as a result push each other unexpectedly. In the event that this took place on the edge of the world, they could be pushed over the wall and out of bounds. With a taller wall we can try prevent them from escaping without some expensive loop making sure that none have left the screen.